review_practice/react-app/src/components/RoomItem/index.js

```diff
@@ -0,0 +1,34 @@
+import React, { useState } from 'react';
+import Style from './style';
+
+const convertOptionDataToStrings = (maxGuest, option) => {
+    const { bed, bedroom, bathroom, freeParking, wifi, kitchen, washer } = option;
+    const optionFirstLine = `인원 ${maxGuest}명 · 침실 ${bedroom}개 · 침대 ${bed}개 · 욕실 ${bathroom}개`;
+    const optionSecondLine = `${freeParking ? '건물 내 무료 주차 ' : ''}${wifi ? '무선 인터넷 ' : ''}${
+        kitchen ? '주방 ' : ''
+    }${washer ? '세탁기 ' : ''}`;
```

Comment:
```suggestion
    const optionSecondLine = [
      freeParking && '건물 내 무료 주차',
      wifi && '무선 인터넷',
      kitchen && '주방',
      washer && '세탁기'
    ].filter(op => !!op).join(' ');
```
로 바꿔보는 것에 대해서 어떻게 생각하시나요?